lib/SGListView.js

```diff
@@ -61,10 +63,81 @@ var SGListView = React.createClass({
         ref='nativeListView'
         renderScrollComponent={this.renderScrollComponent}
         renderRow={this.renderRow}
-        onChangeVisibleRows={this.onChangeVisibleRows} />
+        onScroll={this.handleScroll}
+      />
     );
   },
 
+  /**
+   * Executed on every (throttled) scroll event. I don't like that, but
+   * some quick timing indicates it handles the interval query in <1ms,
+   * so it's definitely not a bottleneck and should perform pretty well
+   * even for a large number of elements.
+   */
+  handleScroll: function(evt) {
+    //  Expand the range that we check so we catch both cells
+    //  that become visible and cells that become invisible:
+    //
+    //  previous interval:  |------------|
+    //                     y1p          y2p
+    //
+    //  current interval:           |------------|
+    //                             y1           y2
+    //
+    //  test interval:      |--------------------|
+    //                     yt1                  yt2
+    //
+    var scrollProperties = this.refs.nativeListView.scrollProperties
+    var y1 = scrollProperties.offset
```

Comment:
`scrollProperties` doesn't have property `offset` Perhaps something changed between `0.8.0` and current?